lab4/src/main/java/edu/csula/web/AdminEventsServlet.java

```diff
@@ -0,0 +1,150 @@
+package edu.csula.web;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.Serializable;
+import java.util.Collection;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import edu.csula.storage.servlet.EventsDAOImpl;
+import edu.csula.storage.EventsDAO;
+import edu.csula.models.Event;
+
+@WebServlet("/admin/events")
+public class AdminEventsServlet extends HttpServlet {
+	
+	private static final long serialVersionUID = -3514842077727662933L;
+
+	private	String events_HTML = "";
+	private	EventsDAOImpl dao;
+	private Collection<Event> events;
+	
+	private final String ADD = "add";
+	private final String EDIT = "edit";
+	private final String DELETE = "delete";
+
+	@Override
+	public void doGet( HttpServletRequest request, HttpServletResponse response) 
+			throws ServletException, IOException {
+		// TODO: render the events page HTML
+		dao = new EventsDAOImpl(getServletContext());
+		events = dao.getAll();
+		response.setContentType("text/html");
+		response.setCharacterEncoding("UTF-8");
+		
+		//DELETE event that was clicked and update event table
+		if (DELETE.equals(request.getParameter("action"))) {
+			int id =  Integer.parseInt(request.getParameter("id"));
+			dao.remove(id);
+			renderEvents(request, response);
+		}
+
+		//RENDER HTML
+		PrintWriter out = response.getWriter();
+			 out.append("<!DOCTYPE html>") 
+				.append("<html lang='eng'>") 
+				.append("<head>") 
+				.append(	"<meta charset='UTF-8'>") 
+				.append(	"<title>admin-events</title>")
+				.append(	"<link rel='stylesheet' href='" + request.getContextPath() + "/app.css'>")
+				.append(	"<h1>")
+				.append(		"World of Coins Framework - Events")
+				.append(	"</h1>")
+				.append("</head>")
+				.append("<nav class='nav'>")
+				.append(	"<a href='admin-info.html'>Information</a> | ")
+				.append(	"<a href='/admin/events'>Events</a> | ")
+				.append(	"<a href='admin-generators.html'>Generators</a>")
+				.append("</nav>")
+				.append("<main>")
+				.append(	"<div class='content'>")
+				.append(		"<div class='forms'>")
+				.append(			"<form action='"+request.getContextPath()+"/admin/events' method='post'>")
+				.append(				"<label>Event ID(USE ONLY IF EDITTING AN EVENT): </label>")
+				.append(				"<input type='number' name='event_id'>")
```

Comment:
It's probably better to keep this id field not editable by either giving `readonly` attribute to the input or make this input to be type `hidden`